app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -390,7 +395,8 @@ def collection_params
           form_class.model_attributes(params[:collection])
         else
           params.permit(collection: {})[:collection]
-                .merge(params.permit(:collection_type_gid))
+                .merge(params.permit(:collection_type_gid)
+                             .with_defaults(collection_type_gid: default_collection_type_gid))
```

Comment:
fixes failure to set collection type in test: `#create` `when setting collection type` `and collection type is not passed`